### ISSUE TITLE
Origin and destination markers A, B

### ISSUE THREAD

**Issue by mahmoudkm** [mapbox/mapbox-directions.js#98]
I've placed new map markers representing point A (origin) and point B (destination) with more app related icons inside my app, but can't figure out how I can remove point A, point B markers from my map or change their icons / color of marker A and B.

So is there any way I can remove point A, B markers or change their color / icons? Thanks

**Comment by wyattp11** [comment]
Yes, you need to modify mapbox.directions.js - search for the markers(you can use their color hex#, size string etc), then you can delete the entire icon object so it doesn't produce A/B icons, just yours will show.
